### ISSUE TITLE
Hidden `b-form-textarea` not auto-sizing when becoming viable

### ISSUE THREAD

**Issue by tradingstratagem** [bootstrap-vue/bootstrap-vue#3936]
### Describe the bug

When a textarea is initially hidden it isn't rendered correctly.
The rows property is not taken into account, so the textarea is very small.

### Steps to reproduce the bug

1. Put a b-form-textarea inside a modal
2. Open the modal
3. The textarea has not the correct height
4. Type in something and it automatically adjust its height

### Expected behavior

The textarea should have the correct height from the beginning

Before
<img width="501" alt="before" src="https://user-images.githubusercontent.com/38089167/63617381-9c30b380-c5e9-11e9-8af8-3229966a9a40.png">

After typing in a character
<img width="506" alt="after" src="https://user-images.githubusercontent.com/38089167/63617382-9c30b380-c5e9-11e9-9ecc-d7ef068be939.png">

### Versions

**Libraries:**

- BootstrapVue: 2.0.0-rc.28
- Bootstrap: 4.3.1
- Vue: 2.6.10

**Environment:**

- Device: Mac
- OS: macOS Mojave
- Browser: Chrome
- Version: 76

**Comment by tmorehouse** [comment]
PR #3937 should have this fixed (once it is merged into 2.0.0 stable branch)

**Comment by tradingstratagem** [comment]
Thank you

**Comment by tmorehouse** [comment]
BootstrapVue v2.0.0 stable has been released. See https://bootstrap-vue.js.org/docs/misc/changelog
